src/corva/models/merge/merge.py

```diff
@@ -25,23 +27,23 @@ class PartialRerunMergeEvent(CorvaBaseEvent):
         run_until: run until
     """
 
-    event_type: EventType
-    partial_well_rerun_id: int
-    rerun_mode: RerunMode
-    start: int
-    end: int
+    event_type: Optional[EventType]
```

Comment:
This is a mandatory field, not optional. They have to send it. If they do not send it this is a problem that should be communicated.